Model/Adminhtml/Observer/AfterShipmentSaveObserver.php

```diff
@@ -126,7 +126,7 @@ public function execute(Observer $observer)
                 $client->setRawData($data, 'application/json');
                 $client->request('POST');
             } catch (\Exception $e) {
-                $this->logger->debug($e->getMessage());
+                $this->logger->debug('Astound\Affirm\Model\Adminhtml\Observer\AfterShipmentSaveObserver::execute', $e->getMessage());
```

Comment:
Can we also try to log the response data if possible?

Reply:
updated here: a7b3513